scripts/dev/upstream_merge/automation_conf.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Is this file still needed? I thought you mentioned that this file will be removed.